src/Uno.UI/UI/Xaml/Controls/FlipView/NativeFlipView.iOS.cs

```diff
@@ -1,14 +1,27 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Windows.UI.Xaml.Media;
 
 namespace Windows.UI.Xaml.Controls
 {
 	public partial class NativeFlipView : PagedCollectionView
 	{
+		private Brush _background;
+
 		public NativeFlipView()
 		{
 			ShowsHorizontalScrollIndicator = false;
 		}
+
+		public Brush Background
+		{
+			get => _background;
+			set
+			{
+				_background = value;
+				BackgroundColor = Brush.GetColorWithOpacity(_background);
+			}
+		}
```

Comment:
This doesn't seem to work as the test is failing. My expectation was that it should work given the template:

https://github.com/unoplatform/uno/blob/ac256db2ceb4da10692f62dcffd644ab80adf54a/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml#L7617